examples/gno.land/r/docs/commondao/commondao.gno

```diff
@@ -0,0 +1,138 @@
+package commondao
+
+import (
+	"std"
+	"strconv"
+
+	"gno.land/p/moul/md"
+	"gno.land/p/moul/mdtable"
+	"gno.land/p/moul/txlink"
+	"gno.land/p/nt/commondao"
+)
+
+const (
+	chaegnal = "g1dzzx5qwcmawqnhhd76ehkteas5v2dg42hqnd7g"
+	leon     = "g125em6arxsnj49vx35f0n0z34putv5ty3376fg5"
+	manfred  = "g1manfred47kzduec920z88wfr64ylksmdcedlf5"
+)
+
+// Creates a DAO with Active and Finished proposal storage
+// Also, it creates a member storage
+// Other options can be added, all of which can be found in 'p/nt/commondao/commondao_options'
+var DAO = commondao.New(
```

Comment:
I think DAO instance should be private, otherwise anyone would be able to call its methods, being able for example to get a proposal reference and there for able to add votes to the voting record, or also accessing the member storage which would allow to add and remove members.